nilearn/signal.py

```diff
@@ -462,18 +493,24 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
                 session_confounds = confounds[sessions == s]
             signals[sessions == s, :] = \
                 clean(signals[sessions == s],
-                      detrend=detrend, standardize=standardize,
+                      detrend=detrend, normalize=normalize,
                       confounds=session_confounds, low_pass=low_pass,
                       high_pass=high_pass, t_r=2.5)
 
-    # detrend
+    if normalize != 'psc':
+        if standardize:
+            normalize = 'std'
+        else:
+            normalize = None
```

Comment:
Here I just wanted to overwrite the value of `normalize` by `standardize` if it is not ''psc", so that output of previous code using `standardize` does not change. `normalize` will work only if it is given value 'psc'